document/FileFormat.mdk

```diff
@@ -2696,31 +2708,385 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `callReturn`     | Recommended: The            | Optional: If the function   | Absent                      |
+|                  | fully qualified name of     | returns a value or          |                             |
+|                  | the  function being         | values, in which case,      |                             |
+|                  | returned from.              | the value or values         |                             |
+|                  |                             | returned.                   |                             |
+|                  |                             | Absent: If the function     |                             |
+|                  |                             | does not return a value.    |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `continuation`   | Absent                      | Absent                      | Optional: The names and     |
+|                  |                             |                             | values of selected          |
+|                  |                             |                             | variables at the specified  |
+|                  |                             |                             | location. Any variable whose|
+|                  |                             |                             | value is available at the   |
+|                  |                             |                             | specified location may be   |
+|                  |                             |                             | mentioned.                  |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `declaration`    | Recommended: The            | Optional: If the            | Optional: If the            |
+|                  | name of the variable        | declaration has an          | declaration has an          |
+|                  | being declared. If          | initializer, in which       | initializer, in which case, |
+|                  | multiple variables are      | case, the value of          | the names and values of     |
+|                  | declared in the same        | the initializer             | any variables               |
+|                  | statement, a separate       | expression.                 | participating in the        |
+|                  | `annotatedCodeLocation`     | Absent: If the              | initializer expression.     |
+|                  | object shall be created     | declaration has no          | Absent: If the              |
+|                  | for each declaration        | initializer.                | declaration has no          |
```

Comment:
Is this strictly true? In .NET, an instance member, if it is a primitive, will receive a default value, even when no initializer is provided. How do we want to handle this?